scripts/validate_task_witness.py

```
@@ -4185,6 +4185,238 @@ def _bridge_source_bytes(root: Path, generation: str, relative: str) -> bytes:
+        ),
+        (
+            b'        "https://github.com/nisavid/agents",\n'
+            b'        "task-witness-smoke",\n',
+            b"        repository,\n"
+            b'        "task-witness-smoke",\n',
+        ),
+        (
+            b"def _plan_intrinsic_smoke_provider(\n"
+            b"    source: CandidateSource,\n"
+            b"    installed_trust_root: Path,\n"
+            b") -> ProviderMaterialization:\n",
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)